x/auth/vesting/types/vesting_account_test.go

```diff
@@ -67,70 +67,98 @@ func (s *VestingAccountTestSuite) SetupTest() {
 
 func TestGetVestedCoinsContVestingAcc(t *testing.T) {
 	now := tmtime.Now()
-	endTime := now.Add(24 * time.Hour)
+	startTime := now.Add(24 * time.Hour)
+	endTime := startTime.Add(24 * time.Hour)
 
 	bacc, origCoins := initBaseAccount()
-	cva, err := types.NewContinuousVestingAccount(bacc, origCoins, now.Unix(), endTime.Unix())
+	cva, err := types.NewContinuousVestingAccount(bacc, origCoins, startTime.Unix(), endTime.Unix())
 	require.NoError(t, err)
 
-	// require no coins vested in the very beginning of the vesting schedule
+	// require no coins vested _before_ the start time of the vesting schedule
 	vestedCoins := cva.GetVestedCoins(now)
 	require.Nil(t, vestedCoins)
 
+	// require no coins vested _before_ the very beginning of the vesting schedule
+	vestedCoins = cva.GetVestedCoins(startTime.Add(-1))
+	require.Nil(t, vestedCoins)
+
 	// require all coins vested at the end of the vesting schedule
 	vestedCoins = cva.GetVestedCoins(endTime)
 	require.Equal(t, origCoins, vestedCoins)
 
 	// require 50% of coins vested
-	vestedCoins = cva.GetVestedCoins(now.Add(12 * time.Hour))
+	vestedCoins = cva.GetVestedCoins(startTime.Add(12 * time.Hour))
 	require.Equal(t, sdk.Coins{sdk.NewInt64Coin(feeDenom, 500), sdk.NewInt64Coin(stakeDenom, 50)}, vestedCoins)
 
+	// require 75% of coins vested
+	vestedCoins = cva.GetVestedCoins(startTime.Add(18 * time.Hour))
```

Comment:
Might be useful and simpler to ready by doing this:
```go
t75 := time.Duration(0.75 * float64(endTime.Sub(startTime)))
vestedAt75Percent := cva.GetVestedCoins(startTime.Add(t75))
```

that'll even make it much easier to test out the linearity of different percentages and that the number of vested coins matches the percentage, rather than by hard coded manually constructed time offsets.